src/index.html

```diff
@@ -5,8 +5,360 @@
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Imac picture"
+          >
+
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="review__amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-block price">
+            <p class="price__title">Price:</p>
+            <p class="price__amount">$2,199</p>
+          </div>
+
+          <a
+            class="card__buy-link"
+            href="/"
+            data-qa="card-hover"
+          >Buy
+          </a>
```

Comment:
Fix  it all over the page
```suggestion
          <a
            class="card__buy-link"
            href="/"
            data-qa="card-hover"
          >
            Buy
          </a>
```